NeoML/src/Dnn/Layers/AccuracyLayer.cpp

```diff
@@ -60,16 +61,17 @@ void CAccuracyLayer::RunOnceAfterReset()
 	CPtr<CDnnBlob> inputBlob = inputBlobs[0];
 	CPtr<CDnnBlob> expectedLabelsBlob = inputBlobs[1];
 	CFastArray<float, 1> inputBuffer;
-	CFastArray<float, 1> expectedBuffer;
-
+	
 	const int dataSize = inputBlob->GetDataSize();
 	const int objectCount = inputBlob->GetObjectCount();
 	const int objectSize = inputBlob->GetObjectSize();
-
 	inputBuffer.SetSize( dataSize );
-	expectedBuffer.SetSize( dataSize );
-	inputBlob->CopyTo( inputBuffer.GetPtr(), dataSize );
-	expectedLabelsBlob->CopyTo( expectedBuffer.GetPtr(), dataSize );
+	inputBlob->CopyTo( inputBuffer.GetPtr(), inputBuffer.Size() );
+
+	const int expectedDataSize = expectedLabelsBlob->GetDataSize();
+	const int expectedObjectSize = expectedLabelsBlob->GetObjectSize();
+	int* expectedBuffer = expectedLabelsBlob->GetBuffer<int>( 0, expectedDataSize );
```

Comment:
Something wrong here with blob data type. Earlier here was the code, which works with `float`. Not it's `int`-only...